### DISCUSSION TITLE
View workflows created in code in Elsa studio

### DISCUSSION THREAD

**Discussion by robverhoef-united** [elsa-workflows/elsa-core#4798] [Q&A]
First off all, thans for a graat library! I am very much a fan and using it activly in more of my projects.
In Elsa V2 it was possible to view a workflow created in Code in the workflow  designer. Although the workflows in code are very much cleaner in Elsa V3 it would stills be very usefull to check the workflow visually. I have not found that option yet in Elsa V3 studio.  Also have a peek at the executed instance of a coded workflow would be very usefull.
Is this already possible or might this be an option to add in a future version?

Thans, Rob

**Comment by sfmskywalker** [comment]
Hey Rob, thanks for the kind words!

In Elsa Studio V3, you can view coded workflows, but the designer currently only supports Flowchart activities for display, and not Sequence activities.

We do intend to add designer support for Sequence activities as well, as described in #3452
Despite no proper designer support for the Sequence activity at the moment, you should still be able to view workflow instances of programmatic workflows and inspect the full Journal.
